Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -522,4 +523,58 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
     }
 
     return frame;
+}
+
+static CGSize _DWriteGetSize(CFAttributedStringRef string, CFRange range, CGSize maxSize, CFRange* fitRange) {
+    CGSize ret = CGSizeZero;
+    if (range.length == 0L) {
+        range.length = CFAttributedStringGetLength(string);
```

Comment:
I'm guessing @msft-Jeyaram missed this part up here wrt the comment below? (thinking that probably at least one of these two deserves a callout in a comment)